scripts/run-in-docker.sh

```diff
@@ -22,22 +22,22 @@ export AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile
 # We are piping to stdin of read command, which lets us set a variable
 # look up read docs - read for linux, cat for mac
 #read -r -d '' SCRIPT <<'EOF'
-# cat <<'EOF'
-# const testPostBody = {
-#     id: "Sally"
-# }
-
-# // This is ignored if we use a GET request
-# const testEvent = { 
-#     httpMethod: POST,
-#     body: JSON.stringify(testPostBody)
-# }
-# console.log(JSON.stringify(testEvent))
-# EOF
-
+SCRIPT=$(cat <<'EOF'
```

Comment:
Removing the quotes here will enable variable substitution in the HEREDOC body.
```suggestion
SCRIPT=$(cat <<EOF
```